ee/clickhouse/views/cohort.py

```diff
@@ -0,0 +1,39 @@
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.queries.util import get_earliest_timestamp
+from ee.clickhouse.sql.person import PERSON_STATIC_COHORT_TABLE
+from posthog.api.cohort import CohortSerializer, CohortViewSet
+from posthog.constants import INSIGHT_STICKINESS, INSIGHT_TRENDS
+from posthog.models.cohort import Cohort
+from posthog.models.entity import Entity
+from posthog.models.filters.filter import Filter
+from posthog.models.filters.stickiness_filter import StickinessFilter
+from posthog.models.team import Team
+from posthog.tasks.calculate_cohort import insert_cohort_from_query
+
+
+class ClickhouseCohortSerializer(CohortSerializer):
+    earliest_timestamp_func = lambda team_id: get_earliest_timestamp(team_id)
+
+    def _handle_stickiness_people(self, cohort: Cohort, filter: StickinessFilter) -> None:
+        insert_cohort_from_query.delay(cohort.pk, INSIGHT_STICKINESS, filter.to_dict())
+
+    def _handle_trend_people(self, cohort: Cohort, filter: Filter) -> None:
+        if len(filter.entities) >= 1:
+            entity = filter.entities[0]
+        else:
+            entity = Entity({"id": filter.target_entity_id, "type": filter.target_entity_type})
+        insert_cohort_from_query.delay(cohort.pk, INSIGHT_TRENDS, filter.to_dict(), entity_data=entity.to_dict())
+
+
+def insert_cohort_people_into_pg(cohort: Cohort):
+    ids = sync_execute(
+        "SELECT person_id FROM {} where team_id = %(team_id)s AND cohort_id = %(cohort_id)s".format(
+            PERSON_STATIC_COHORT_TABLE
```

Comment:
addressed on call but leaving notes here. We need the table to be within clickhouse so that querying by static cohorts can filter on the people in the cohort. This was set up when creating functionality for cohorts by CSV